documentation/marketing_usage.md

```diff
@@ -0,0 +1,112 @@
+
+```text
+Welcome {{ full_name }}!
+
+Your Open edX instance {{ instance_name }} is ready!
+
+Thank you for signing up with OpenCraft!
+We are very excited to have you onboard.
+
+If you need anything, or have any questions or suggestions,
+don’t hesitate to reply to this email.
+```
+
+#### Advanced template features
+
+Two reusable templates defined in OCIM source can also be used in your Email Templates
+
+1. `emails/_email_signature.html`: This template can be used to add the default signature to the emails. Add `{% include “emails/_email_signature.html %}`  in any html body where you wish to include the default signature.
+2. `emails/email_base.html`: This template contains the default styling of an OpenCraft email and can be used as base for the html body of your email templates (see image below). Add ``{% extends "emails/email_base.html" %}`` at the beginning of html body and include the content inside as `{% block content %} <your content here> {% endblock %}`
+
+The following screenshot represents these special email components:
+
+![Advanced email example](images/advanced-email-template.jpg)
+
+Following is a sample email template using the advanced features:
+
+```html
+{% extends "emails/email_base.html" %}
+{% block content %}
+    <p>Hello {{ full_name }},</p>
+    <p>Click the following link to receive a fresh pizza for free every week:
+      <a href="https://pizza.com">https://pizza.com</a>
+    </p>
+
+    <p>Thanks for your consideration.</p>
+
+    <p>Best,&nbsp;</p>
+    <p>Joe Pizza</p>
+
+    {% include "emails/_email_signature.html" %}
+{% endblock %}
+```
+
+### Testing the email templates
+
+Make sure to test the email templates before marking them active and actually sending them to our users. Following are the instructions to test them before activating:
+
+1. Open email templates in admin ( admin/marketing/emailtemplate/ )
+2. Select the email template to test.
+3. Select `Send sample email for selected templates` from the action dropdown at the bottom of the page and hit the Go button.
+![Send Sample Toolbar](images/send-sample-toolbar.png)
+
+You should receive an email in your inbox shortly with some dummy data populated for the application.
+
+## Subscribers
+
+Subscribers are a list of users who will receive the followup emails. A user not added to the Subscriber list will not receive any emails.
+
+### Adding a Subscriber
+
+A user is automatically added to the subscriber list when the first appserver is provisioned and made active.
+
+To add a new subscriber manually
+
+1. Visit /admin/marketing/subscriber/add/ on the OCIM site
+2. Fill in the user details
+3. Make sure to check the “Receive followup” checkbox. You can uncheck this field later to unsubscribe a user.
+4. Click on Save button to save the changes.
+
+!!! note
+
+	1. Emails are triggered everyday at 0100 hrs UTC and a report is sent to `billing@opencraft.com`, containing a list of all the emails sent afterwards.
```

Comment:
@arjunsinghy96, I think we shouldn't add our internal email id in our OpenSource doc like this. Rather mentioning it will send report to `MARKETING_EMAIL_REPORT_RECIPIENTS` would be better. What do you think?

Reply:
Thanks for pointing this out @shimulch .